tests/nightly/test_tutorial_config.txt

```diff
@@ -1,20 +1,31 @@
 basic/ndarray
+basic/ndarray_indexing
 basic/symbol
 basic/module
 basic/data
-python/linear-regression
-python/mnist
-python/predict_image
-onnx/super_resolution
-onnx/fine_tuning_gluon
-onnx/inference_on_onnx_model
-basic/ndarray_indexing
-python/matrix_factorization
+gluon/customop
+gluon/data_augmentation
+gluon/datasets
 gluon/ndarray
 gluon/mnist
 gluon/autograd
 gluon/gluon
 gluon/hybrid
+nlp/cnn
+onnx/super_resolution
+onnx/fine_tuning_gluon
+onnx/inference_on_onnx_model
+python/matrix_factorization
+python/linear-regression
+python/mnist
+python/predict_image
+python/data_augmentation
+python/data_augmentation_with_masks
+python/kvstore
+python/types_of_data_augmentation
 sparse/row_sparse
 sparse/csr
-sparse/train
```

Comment:
Can we pick them automatically so we don't need to keep updating this file?

Reply:
Indeed, going forward, there will be a one individual test per tutorial, to allow the use of annotation like `@highCpu`, `@hihMemory`. And there will be an integration test that will check that each notebook has been added to the test suite. 

This will be part of my next PR, as part of this work of integrating tutorials to the CI 😃

Reply:
Yeah, sorry @rahul003. Thomas suggested the same as you, but we will have to be able to annotate the tests and give the possibility to add custom testing behaviour - e.g. methods to validate a tutorial which are different for each one.